hotsos/core/plugins/kernel/memory.py

```diff
@@ -63,6 +65,7 @@ def path(self):
         return os.path.join(HotSOSConfig.data_root, 'proc/vmstat')
 
     @property
+    @alias("kernel.vmstat.compaction_failures_pct")
```

Comment:
I think this isn't needed since the class is aliased. Can be directly used as `kernel.vmstat.compaction_failures_percent`.

Reply:
nice catch, I'll remove it